framework/PostProcessors/BasicStatistics.py

```diff
@@ -314,6 +318,9 @@ def _handleInput(self, paramInput):
       elif tag == "dataset":
         if child.value.lower() in utils.stringsThatMeanTrue():
           self.outputDataset = True
+      elif tag == "multipleLinear":
+        if child.value.lower() in utils.stringsThatMeanFalse():
```

Comment:
If you follow what I wrote just above, this if statement is not needed

Reply:
fixed